components/scripts/gradle/gradle-init-scripts/configure-gradle-enterprise.gradle

```diff
@@ -87,6 +86,11 @@ if (ccudPluginVersion && isNotAtLeast(ccudPluginVersion, '1.7')) {
 def registerBuildScanActions = { def buildScan ->
     def scanFile = new File(expDir, 'build-scans.csv')
     buildScan.buildScanPublished { publishedBuildScan ->
+        def getInputParamCompatibleWithCC = { String name ->
```

Comment:
Add a comment to this closure explaining that we defer reading the `runNum` system property to the execution time since that property does not affect the configuration of the build but changes between multiple build invocations. Reading a changing system property value at configuration time in a way that it is not tracked as an input of the configuration model (which we cannot guarantee due to the -XX cases) would mean that for configuration cache-compatible builds, the value determined in the first build would be reused in the second build.